handbook/engineering/cloud/goals.md

```diff
@@ -0,0 +1,69 @@
+# Cloud goals and priorities
+
+## Guiding principles
```

Comment:
Any objection to me moving this up one level for all of engineering, not just cloud team?

Reply:
No objection at all :)